src/test/java/teammates/test/cases/logic/StudentsLogicTest.java

```diff
@@ -680,7 +681,7 @@ private void testEnrollStudents() throws Exception {
                 .withEndTime(TimeHelperExtension.getInstantHoursOffsetFromNow(5))
                 .withSessionVisibleFromTime(TimeHelperExtension.getInstantHoursOffsetFromNow(1))
                 .withResultsVisibleFromTime(TimeHelperExtension.getInstantHoursOffsetFromNow(6))
-                .withTimeZone(8)
+                .withTimeZone(TimeHelper.convertToZoneId(8))
```

Comment:
Just use something like `ZoneId.of("+08:00")`? `convertToZoneId` should be removed eventually.